### ISSUE TITLE
Feature request - Speak on demand

### ISSUE THREAD

**Issue by TomVal** [nvaccess/nvda#5948]
My eyes aren't good at all, but I still can use them. I'd like to save them with help of NVDA. I don't need automatic spoken feedback when focus changes, but I'd like still to use various NVDA commands without having to constantly toggle speech off and on. So it would be nice, if it was possible to set speech mode to "speak on demand", where automatic feedback on focus change would be muted, but spoken feedback would occur whenever appropriate NVDA command was issued, e.g. NVDA+F12. It would also help in cases when an app doesn't work well with NVDA, giving erratic spoken feedback, which apps I can still use and control visually and/or using built-in Windows mangifier.

**Comment by ehollig** [comment]
Duplicate of #481
